doc/api/events.md

```diff
@@ -1361,6 +1361,32 @@ changes:
       The `EventTarget` class is now available through the global object.
 -->
 
+### `EventTarget.maxListenersSymbol`
```

Comment:
This symbol needs to be someplace else that's not on `EventTarget` as a static.
I checked the spec and asked Domenic and this is not allowed.

This can be a static method on `events` called `setMaxListeners` that works for both EventTarget and EventEmitter for example (that's the approach I took here):  https://github.com/nodejs/node/pull/35991

Reply:
Ok, I added the symbol as `require('events').maxEventTargetListenersSymbol`, to be called as...

```js
const et = new EventTarget();
et[events.maxEventTargetListenersSymbol](5);
```